### PR TITLE
writing lab fixes

### PR DESCRIPTION
**Pull request by tram**

### CODE REVIEW

File: data_capture/templates/data_capture/price_list/step_5.html [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 {% block step_body %}
 <h2>Success!</h2>
-<p>Your data has been submitted for review by a CALC admin. It should be available soon!</p>
+<p>You’ve submitted your data for review — it should be available soon!</p>
```
Comment (toolness) [inline data_capture/templates/data_capture/price_list/step_5.html]: Huh—I thought there was supposed to be no spaces around em dashes, but perhaps I am mistaken.

File: data_capture/tests/test_price_list_views.py [hunk 2]
```diff
@@ -295,7 +295,7 @@ def test_invalid_post_via_xhr_returns_json(self):
         self.assertHasMessage(
            res,
            'error',
-           'Oops, please correct the error below and try again.'
+           'Oops! Please correct the following error.'
```
Comment (toolness) [inline data_capture/tests/test_price_list_views.py]: Heh, I guess in the future we should move this to a named constant like `ERROR_TEXT` so we don't have to do all this find/replacing whenever we decide to change the copy!

**Comment by toolness** [comment]
This looks great so far! 💃 🎉 🍷 🇺🇸

File: data_capture/templates/data_capture/price_list/step_5.html (hunk 1 above)
Comment (tram) [inline data_capture/templates/data_capture/price_list/step_5.html]: I thought the same thing. Let's do it.

File: data_capture/templates/data_capture/price_list/step_5.html [hunk 3]
```diff
@@ -4,5 +4,5 @@
 
 {% block step_body %}
 <h2>Success!</h2>
-<p>You’ve submitted your data for review — it should be available soon!</p>
+<p>You’ve submitted your data for review&#8212;it should be available soon!</p>
```
Comment (toolness) [inline data_capture/templates/data_capture/price_list/step_5.html]: A slightly more readable version of this might be `&mdash;` but it is up to you.

Comment (tram) [inline data_capture/templates/data_capture/price_list/step_5.html]: Good point! Changing it… :-)

**Comment by toolness** [comment]
This looks great!  Feel free to address my one comment if you agree with it (no worries if not) and merge (by clicking the "Merge pull request" button) whenever you want.

**Comment by jseppi** [comment]
🎉 🎆 🚀

**Pull request merged**